src/wallet/wallet.cpp

```diff
@@ -1344,6 +1344,22 @@ void CWalletTx::GetAmounts(list<COutputEntry>& listReceived,
         nFee = nDebit - nValueOut;
```

Comment:
Shouldn't the fee calculation take into account the adjustment to the transparent value pool from JoinSplits?